pkgs/tools/misc/most/default.nix

```
@@ -1,11 +1,12 @@
 { stdenv, fetchurl, slang, ncurses }:
 
-stdenv.mkDerivation {
-  name = "most-5.0.0a";
+stdenv.mkDerivation rec {
+  pname = "most";
+  version = "5.1.0";
 
```

```suggestion

  outputs = [ "out" "doc" ];

```

has stuff at `share/doc` that aren't rather useful if installed.

---

probably want to do
```
makeFlags = [
  "DOC_DIR=${placeholder ''doc''}/share/doc/most"
];
```

so they can just get installed there.

##### found after inspecting makefiles